src/modules/75-cf/components/CreateFlagWizard/FlagElemAbout.module.scss.d.ts

```diff
@@ -7,8 +7,7 @@
  **/
 // this is an auto-generated file, do not update this manually
 declare const styles: {
-  readonly aboutFlagContainer: string
-  readonly checkboxIcon: string
+  readonly footerBtns: string
   readonly tooltipAboutFlag: string
```

Comment:
```suggestion
  readonly aboutFlagTooltip: string
```

NBD: But I feel this reads better?

Reply:
I didn't write this but can update it if you want